### PR TITLE
Add calls

### PR DESCRIPTION
**Pull request by NiallRees**
Waiting on https://github.com/facetoe/zenpy/pull/454

### CODE REVIEW

File: tap_zendesk/schemas/calls.json [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+                "null",
+                "boolean"
+            ]
+        },
+        "phone_number": {
+            "type": [
+                "null",
+                "string"
+            ]
+        },
```
Comment (NiallRees) [inline tap_zendesk/schemas/calls.json:181]:
```suggestion
        },
        "phone_number_id": {
            "type": [
                "null",
                "integer"
            ]
        },
```

Waiting on upstream: https://github.com/facetoe/zenpy/pull/454#issuecomment-710077200

**Comment by kgpayne** [comment]
@NiallRees and @koszti (hey 👋) - please can we push this forward? Zenpy has merged Nialls upstream change (in 2.0.23). It may be good to upgrade to 2.0.24 to close the dependabot pr #13 at the same time 🙂🙏

**Comment by kgpayne** [comment]
Actually, this pr isn't quite ready 😅 I have an updated fork with this implementation [here](https://github.com/tailsdotcom/pipelinewise-tap-zendesk). @NiallRees (assuming this is no longer important to you) do you want to close this pr and I'll make another from the tails fork of your fork?

**Comment by NiallRees** [comment]
> Actually, this pr isn't quite ready 😅 I have an updated fork with this implementation [here](https://github.com/tailsdotcom/pipelinewise-tap-zendesk). @NiallRees (assuming this is no longer important to you) do you want to close this pr and I'll make another from the tails fork of your fork?

Go for it.

**Comment by koszti** [comment]
@kgpayne, sure if you send PR I'm happy to merge.

**Pull request closed without merging**